forge-gui/res/cardsfolder/upcoming/ulder_ravengard_marshal.txt

```diff
@@ -0,0 +1,11 @@
+Name:Ulder Ravengard, Marshal
+ManaCost:1 R W
+Types:Legendary Creature Human Noble Soldier
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPumpEOT | TriggerDescription$ When CARDNAME enters the battlefield, another target nontoken creature you control gains double team until end of turn.
+SVar:TrigPumpEOT:DB$ Pump | ValidTgts$ Creature.YouCtrl+nonToken+Other | TgtPrompt$ Select another target nontoken creature | KW$ Double Team
```

Comment:
`Select another target nontoken creature` > `Select another target nontoken creature you control`